### DISCUSSION TITLE
Zig build support android platform

### DISCUSSION THREAD

**Discussion by meiyese** [jedisct1/libsodium#1370] [Q&A]
Hi,

we use zig to build for multiple platforms, I'd like to know that if it is possible to use zig build for android platform?

**Comment by jedisct1** [comment]
Hi,

It's definitely possible, but requires the Android SDK, and this is not currently supported in libsodium. I can look at it, but my knowledge of Android is fairly limited.

So, the way to compile libsodium for Android remains the `./dist-build/android-aar.sh` script.

**Reply by meiyese** [reply]
Thank you for clarifying. I will continue to use `./dist-build/android-aar.sh` script. Zig is really convenient. Looking forward to having the opportunity to use it. Thanks for your awesome work.

**Comment by kasperisager** [comment]
For reference, this almost works: https://github.com/ziglang/zig/issues/12114#issuecomment-2697769570
